pantheon-decoupled-gatsby.php

```diff
@@ -16,7 +16,6 @@
 
 function pantheon_decoupled_gatsby_activate() {
   if (get_option('pantheon_decoupled_gatsby_activated') !== 1) {
-    activate_plugin('pantheon-decoupled/pantheon-decoupled.php');
```

Comment:
Did keeping this cause any specific problems on install? Does activate plugin throw an error if the module is already installed?

Conceptually I think keeping this makes sense - that way if someone installs this plugin first, pantheon-decoupled will be installed automatically.

Reply:
I didn't see any consequence for on doing so.